frontend/src/components/Room/RoomComponent.js

```diff
@@ -47,23 +67,17 @@ const RoomComponent = ({ roomList, uuid }) => {
                                             type="danger"
                                             htmlType="submit"
                                             onClick={() => deleteRoom(key)}
-                                            className="btn btn-sm"
+                                            className="btn btn-sm delete-btn"
```

Comment:
```suggestion
                                            className="btn btn-sm delete-btn"
```

please delete unused className